### PR TITLE
Updating Subscriber's Channel Status based on PullSubscription Status

### LINKED ISSUE #179: read the condition from PullSubscription, and add a message if it's not ready.
read the condition from PullSubscription, and add a message if it's not ready.

_Originally posted by @n3wscott in https://github.com/google/knative-gcp/pull/178/files#r314812446_

### PR DESCRIPTION
**Pull request by nachocano**
Fixes #179 

- Synchronizing subscriber statuses using PullSubscription statuses.

### CODE REVIEW

File: pkg/reconciler/channel/channel.go [hunk 1]
```diff
@@ -366,7 +394,7 @@ func (r *Reconciler) getTopic(ctx context.Context, channel *v1alpha1.Channel) (*
 	}
 	if !metav1.IsControlledBy(topic, channel) {
 		channel.Status.MarkTopicNotOwned("Topic %q is owned by another resource.", name)
-		return nil, fmt.Errorf("Channel: %s does not own Topic: %s", channel.Name, name)
+		return nil, fmt.Errorf("channel: %s does not own Topic: %s", channel.Name, name)
```
Comment (n3wscott) [inline pkg/reconciler/channel/channel.go]: Channel is the kind and has a cap C

**Comment by n3wscott** [comment]
/lgtm
/hold
/Approve
The Channel comment is the hold.

File: pkg/reconciler/channel/channel.go (hunk 1 above)
Comment (nachocano) [reply pkg/reconciler/channel/channel.go]: I think errors should start with lowercase, but I'm OK, done the change

**Pull request merged**